limitador-server/docs/http_server_spec.json

```diff
@@ -2,6 +2,7 @@
   "swagger": "2.0",
   "definitions": {
     "CheckAndReportInfo": {
+      "type": "object",
```

Comment:
@unleashed just in case you are not aware, this is generated automatically using https://github.com/wafflespeanut/paperclip